File: LC/C++/Easy/3442.cpp
class Solution {
public:
    int maxDifference(string s) {
        std::unordered_map<char, int> count;
        for (char c : s) {
            count[c]++;
        }

        int a1 = 0;
        int a2 = INT_MAX;
        
        for (const auto& [ch, v]: count) {
            if (v % 2 == 1) {
                a1 = std::max(a1, v);
            } else {
                a2 = std::min(a2, v);
            }
        }

        return a1 - a2;
    }
};

File: LC/Python/Easy/3442.py
class Solution:
    def maxDifference(self, s: str) -> int:
        count = Counter(s)

        a1 = 0
        a2 = 100
        for v in count.values():
            if v % 2 == 1:
                a1 = max(a1, v)
            else:
                a2 = min(a2, v)

        return a1 - a2